src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -230,7 +229,7 @@ public class ChartEditorPresenter : AbstractCommandCollectorPresenter<IChartEdit
          _dataBrowserPresenter.SelectionChanged += (o, e) => onSelectionChanged(e.Columns);
 
          _chartExportSettingsPresenter.ChartExportSettingsChanged += (o,e)=> onChartPropertiesChanged();
-         _chartSettingsPresenter.ChartSettingsChanged += (o, e) => onChartPropertiesChanged();
+         _chartSettingsPresenter.ChartSettingsChanged += (o, e) => updateChart();
```

Comment:
Need to call full update when settings are changing (full update redraws everything). Indeed, sideMarginEnabled updates axis and not only chart